CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contribution Guidelines
+
+## Introduction
+
+This document explains how to contribute changes to the Gitea project.
+It assumes you have followed the
+[installation instructions](https://github.com/go-gitea/docs/tree/master/en-US/installation)
+
+Sensitive security-related issues should be reported to [security@gitea.io](mailto:security@gitea.io).
+
+## Discuss your design
+
+The project welcomes submissions but please let everyone know what
+you're working on if you want to change or add to the Gitea repositories.
+
+Before undertaking to write something new for the Gitea project,
+please [file an issue](https://github.com/go-gitea/gitea/issues/new).
+Significant changes must go through the
+[change proposal process](https://github.com/go-gitea/proposals)
+before they can be accepted.
+
+This process gives everyone a chance to validate the design,
+helps prevent duplication of effort,
+and ensures that the idea fits inside the goals for the project and tools.
+It also checks that the design is sound before code is written;
+the code review tool is not the place for high-level discussions.
+
+## Testing redux
+
+Before sending code out for review, run all the tests for the whole
+tree to make sure the changes don't break other usage and keep the compitable when upgrade:
+
+After running for a while, the command should print
+
+```
+ALL TESTS PASSED
+```
+
+## Code review
+
+Changes to Gitea must be reviewed before they are accepted,
+no matter who makes the change even if you are the owners or maintainers.
+We use github's pull request workflow to do that and use lgtm to keep every PR has more than 2 maintainers to reviewed.
```

Comment:
... to ensure every PR is reviewed by at least 2 maintainers. 

(Abd turn lgtm to a link maybe)